src/lang/LanguageManager.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\lang;
+
+use function array_filter;
+use function array_map;
+use function explode;
+use function file_exists;
+use function is_dir;
+use function parse_ini_file;
+use function scandir;
+use function substr;
+use function is_array;
+use const INI_SCANNER_RAW;
+use const SCANDIR_SORT_NONE;
+
+class LanguageManager
+{
+	/** @var Language[] */
+	private $languages = [];
+
+	/** @var string */
+	public const FALLBACK_LANGUAGE = 'eng';
+
+	/** @var string[] */
+	public static $localeToLangCode = [
+		'bg_BG' => 'bul',
+		'cs_CZ' => 'ces',
+		'da_DK' => 'dan',
+		'de_DE' => 'deu',
+		'el_GR' => 'eli',
+		'en_GB' => 'eng',
+		'en_US' => 'eng',
+		'es_ES' => 'spa',
+		'es_MX' => 'spa',
+		'fi_FI' => 'fin',
+		'fr_CA' => 'fra',
+		'fr_FR' => 'fra',
+		'hu_HU' => 'hun',
+		'id_ID' => 'ind',
+		'it_IT' => 'ita',
+		'ja_JP' => 'jpn',
+		'ko_KR' => 'kor',
+		'nb_NO' => 'nor',
+		'nl_NL' => 'nld',
+		'pl_PL' => 'pol',
+		'pt_BR' => 'por',
+		'pt_PT' => 'por',
+		'ru_RU' => 'rus',
+		'sk_SK' => 'slv',
+		'sv_SE' => 'swe',
+		'tr_TR' => 'tur',
+		'uk_UA' => 'ukr',
+		'zh_CN' => 'chs',
+		'zh_TW' => 'zho'
+	];
+
+	public function __construct(string $path)
+	{
```

Comment:
Formatting: brace indentation should match the rest of PocketMine's classes

Reply:
This is not applicable you can fix it

Reply:
Code style is one of the primary concerns when deciding whether to merge a pull request. The responsibility of correcting code style is at the pull request author not the maintainer.